src/actions/actions.ts

```diff
@@ -2291,6 +2290,14 @@ abstract class MoveInsideCharacter extends BaseMovement {
       stop  : endPos,
     };
   }
+
+  public async execActionForOperator(position: Position, vimState: VimState): Promise<IMovement> {
+    const res = await this.execAction(position, vimState);
+
+    res.stop = res.stop.getRight();
```

Comment:
Sorry I forget to mention that if the movment can be across lines, we should use `getRightThroughLineBreaks` and I think `ci(` can be operated on multiple lines, right?